### PR TITLE
DPC-905: Set up env variables for static site separation

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-905](https://jiraent.cms.gov/browse/DPC-905) & [DPC-1122](https://jiraent.cms.gov/browse/DPC-1122)

For easier navigation, set up variables for production and sandbox registration and sign in links.

### Proposed Changes

- Created the four variables `prod_register`, `prod_sign_in`, `sbx_register`, and `sbx_sign_in`
- Replaced missing links for sandbox registration on home and FAQ pages

### Change Details

- Changed nav links to register/sign in direct to sandbox web app.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### Acceptance Validation

Screen Shot 2020-12-03 at 2 08 03 PM

:white_check_mark:  Update the existing PR with master, and resolve any merge conflicts
:white_check_mark:  Add a redirect for the old login URL (https://dpc.cms.gov/users/sign_in) to the new login URL (https://sandbox.dpc.cms.gov/users/sign_in)

Screen Shot 2021-02-18 at 3 41 19 PM

:white_check_mark:  Publish the static site branch (nz/dpc-905-url-var) to stage.dpc.cms.gov
:white_check_mark:  Verify that all content works, and that all links to the web app point to the expected sandbox URLs

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -28,6 +28,10 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://dpc.cms.gov" # the base hostname & protocol for your site, e.g. http://example.com
 cssurl: /assets/css/static-main.css
 logourl: /assets/images/top-nav-heart.svg
+prod_register: "https://prod.dpc.cms.gov/users/sign_up"
```
Comment (dhgreene) [inline _config.yml:31]: It's generally an anti-pattern to define variables (or in this case config params) we're not using.  In particular, I'm not sure yet whether we'll want a signup link in production, so I question including this one.

**Review by dhgreene — APPROVED** [review]
Thanks for catching the other links!

File: _config.yml (hunk 1 above)
Comment (Sun-Mountain) [reply _config.yml:31]: It's easy enough to remove and then add again. You'll have to approve the PR again.

Comment (Sun-Mountain) [reply _config.yml]: @dhgreene Please approve again! 🙏

**Review by dhgreene — APPROVED** [review]
👍

**Pull request merged**